temescal/__init__.py

```diff
@@ -199,7 +199,7 @@ def test_tone(self):
         self.send_packet(data)
 
     def set_night_mode(self, enable):
-        data = {"cmd": "set", "data": {"b_night_mode": enable}, "msg": "SETTING_VIEW_INFO"}
+        data = {"cmd": "set", "data": {"b_night_time": enable}, "msg": "SETTING_VIEW_INFO"}
```

Comment:
Hm, mine is definitely b_night_mode. That's annoying. One way of handling this would be to query the state first and then see which option is provided?

Reply:
Oh ok, should i try to implement this? Maybe this should be used for feature detection as well like rear/center/surround speaker, ...?